src/NServiceBus.RabbitMQ.sln

```diff
@@ -37,4 +37,7 @@ Global
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {0AA8D0E5-66A3-4FEA-BC1F-013150CFA8F8}
+	EndGlobalSection
```

Comment:
It's something that's been showing up when the latest VS preview edits a solution file. I'm not entirely sure what is, but it insists on it being there.